workflow/executor/executor.go

```diff
@@ -931,6 +958,100 @@ func (we *WorkflowExecutor) Wait(ctx context.Context) error {
 	return nil
 }
 
+var progressRegexp = regexp.MustCompile("#argo .*progress=([0-9]+/[0-9]+)")
```

Comment:
I think we should only allow the file to contain the progress as `N/M` format

Reply:
you mean just writing `N/M` into the file without the whole `#argo progress` decoration? 
I guess that'd work yes.

Reply:
simpler